tests/pkg_micro-ecc/tests/01-run.py

```diff
@@ -4,16 +4,18 @@
 from testrunner import run
 
 
-# Use a custom global timeout for slow hardware. On microbit (nrf51), the
-# test completes in 80s.
-TIMEOUT = 100
+# Use a custom global timeout for slow hardware. On ATmegas clocked at 8MHz
+# test completes in ~10 min
+TIMEOUT = 600
 
 
 def testfunc(child):
     child.expect_exact('micro-ecc compiled!')
     child.expect_exact('Testing 16 random private key pairs and signature '
                        'without using HWRNG')
-    child.expect_exact('................ done with 0 error(s)')
+    for i in range(15):
```

Comment:
Small improvement: the value of 15 could be determined via the line before:
```py
child.expect(r'Testing (\d+) random private key pairs and signature '
             'without using HWRNG')
testrounds = child.match.group(1)
for i in range(testrounds - 1):
```
This would also allow to override the `TESTROUNDS` macro via CFLAGS and the automatic test would still work.

Reply:
Excellent idea. (I have a typo in the `range()`, it will expect one round less. I'll fix that as well.)

Reply:
Done